app.rb

```diff
@@ -0,0 +1,119 @@
+require_relative 'classes/book'
+require_relative 'classes/label'
+require_relative 'data_persist/book_persist'
+require_relative 'data_persist/label_persist'
+require_relative 'data_persist/handle_json'
+require_relative 'factory'
+require_relative 'classes/musicalbum'
+require_relative 'classes/genre'
+require_relative 'data_persist/music_persist'
+require_relative 'data_persist/genre_persist'
+require_relative 'classes/author'
+require_relative 'classes/game'
+require_relative 'data_persist/author_persist'
+require_relative 'data_persist/game_persist'
+
+class App
+  def initialize
+    @books = []
+    @labels = []
+    @music_album = []
+    @genres = []
+    @authors = []
+    @games = []
+
+    load_books
+    load_labels
+    load_author
+    load_game
+    load_genre
+    load_music
+  end
+  include MethodesFactory
+  include BookPersist
+  include HandlerFile
+  include LabelPersist
+  include MusicPersist
+  include GenrePersist
+  include GamePersist
+  include AuthorPersist
+
+  def take_input(label)
+    print "#{label}: "
+    gets.chomp
+  end
+
+  def add_book
+    publisher = take_input('Publisher')
+    publish_date = take_input('Publish_date')
+    puts 'Enter Good or Bad for the cover state'
+    cover_state = take_input('Cover_state')
+    puts 'Is the book archived? (true/false)'
+    archived = take_input('Archived')
+    insert_book(publisher, publish_date, cover_state, archived)
+  end
+
+  def list_books
+    @books.each do |book|
+      puts "[#{book.class}] Publisher: #{book.publisher} Publish_date: #{book.publisher} Cover_state: #{book.cover_state} Archived: #{book.archived}"
+    end
+  end
+
+  def add_label
+    title = take_input('Title')
+    color = take_input('Color')
+    insert_label(title, color)
+  end
+
+  def list_labels
+    @labels.each do |label|
+      puts "[#{label.class}] Title: #{label.title} Color: #{label.color}"
+    end
+  end
+
+  def add_music
+    name = take_input('Name')
+    publish_date = take_input('Publish_date')
+    on_spotify = take_input('On_spotify?')
+    insert_music(name, publish_date, on_spotify)
+  end
+
+  def list_music
+    @music_Album.each do |music|
+      puts "[#{music.class}] Name: #{music.name} Publish_date: #{music.publish_date} On_spotify: #{music.on_spotify}"
+    end
+  end
```

Comment:
- [ ] I am getting errors when try sees music albums. Kindly fix it.

![image](https://user-images.githubusercontent.com/38635158/189489757-48af1c62-78b7-4a4a-ae1c-c8f0ba6f6861.png)